packages/ui-app/src/Button/types.d.ts

```diff
@@ -18,6 +18,7 @@ export type ButtonProps = BareProps & {
   isPrimary?: boolean,
   onClick?: () => void | Promise<void>,
   size?: Button$Sizes,
+  tabIndex?: number | string,
```

Comment:
Only `number`?

Reply:
If I change it from `tabIndex?: number | string` to `tabIndex?: number`, it causes linting errors.

```
tabIndex?: number
         ~~~~~~~~
    The expected type comes from property 'tabIndex' which is declared here on type 'IntrinsicAttributes & BareProps & { children?: ReactNode; floated?: "left" | "right" | undefined; icon?: string | undefined; isBasic?: boolean | undefined; isCircular?: boolean | undefined; ... 7 more ...; tabIndex?: number | undefined; } & { ...; }'
```

Because in Input.tsx https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/Input.tsx#L8 it uses `import SUIInput from 'semantic-ui-react/dist/commonjs/elements/Input/Input';`, where it's defined as 

```
  /** An Input can receive focus. */
  tabIndex?: number | string
```

Here's the line of code in their latest version, which still allows number or string https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/elements/Input/Input.js#L91